docs/concepts/tasks.md

```diff
@@ -204,29 +204,80 @@ def my_flow():
 
 ## Retries
 
-Prefect tasks can automatically retry on failure. To enable retries, pass `retries` and `retry_delay_seconds` parameters to your task.
+Prefect can automatically retry tasks on failure. In Prefect, a task _fails_ 
+if it returns a value that is not a `State` object, or if it raises an
+exception.
 
-For example, let's say you need to retrieve data from a brittle API:
+To enable retries, pass `retries` and `retry_delay_seconds` parameters to your
+task. If the task fails, Prefect will retry it up to `retries` times, waiting
+`retry_delay_seconds` seconds between each attempt. If the task fails on the
+final retry, Prefect marks the task as _crashed_ if the task raised an exception
+or _failed_ if it returned a string.
+
+!!! note "Retries don't create new task runs"
+    A new task run is not created when a task is retried. A new state is added to the state history of the original task run.
+
+### A simple example: dividing by zero
+
+For example, this task divides a numerator by a random denominator without 
+checking to ensure that `denominator` is a non-zero number:
+
+```python
+from random import randint
+from prefect import task
+
+@task(retries=3)
```

Comment:
This doesn't work, right? A task must be in a flow.

Reply:
```RuntimeError: Tasks cannot be run outside of a flow. To call the underlying task function outside of a flow use task.fn()
```

Reply:
This is the definition of a task. You have to call or submit tasks from a flow.

Reply:
Ah I see, you didn't intend for this to be a runnable example. Most (all?) of the other code examples that we include in the docs are runnable, so including a code snippet that can't be run in isolation is a departure from the norm. If we are going to include an example that isn't runnable in isolation, I think we should remove the `import` statements, so it doesn't look like its code meant to be copied, pasted, and run in isolation.

Reply:
We also might want to explicitly call out thats its just a definition, and must be called from a flow.

Reply:
Ah, gotcha. I see. The other examples in this section aren't runnable, so I didn't even notice! I'll adapt this to have a flow that calls it.